<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tab7control</class>
 <widget class="QWidget" name="tab7control">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>543</width>
     <height>408</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>541</width>
      <height>406</height>
     </rect>
    </property>
    <widget class="QPushButton" name="pPBRed">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>40</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="IotQtClient.qrc">
       <normaloff>:/IotQtClient/Image/red_off.png</normaloff>
       <normalon>:/IotQtClient/Image/red_on.png</normalon>:/IotQtClient/Image/red_off.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pPBBlue">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>130</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="IotQtClient.qrc">
       <normaloff>:/IotQtClient/Image/blue_off.png</normaloff>
       <normalon>:/IotQtClient/Image/blue_on.png</normalon>:/IotQtClient/Image/blue_off.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pPBPurple">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>220</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="IotQtClient.qrc">
       <normaloff>:/IotQtClient/Image/purple_off.png</normaloff>
       <normalon>:/IotQtClient/Image/purple_on.png</normalon>:/IotQtClient/Image/purple_off.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="IotQtClient.qrc"/>
 </resources>
 <connections/>
</ui>
